connectors/src/lib/models/slack.ts

```diff
@@ -21,6 +21,7 @@ export class SlackConfigurationModel extends Model<
   declare botEnabled: boolean;
   declare connectorId: ForeignKey<ConnectorModel["id"]>;
   declare whitelistedDomains?: readonly string[];
+  declare whiteListedChannelPattern: string | null;
```

Comment:
To align on the DB definition with no default value, the type should include `undefined`.
```suggestion
  declare whiteListedChannelPattern?: string | null;
```